src/api/services/adminServiceHelpers.ts

```diff
@@ -57,11 +57,23 @@ export const AllowedSiteRoles: Record<string, AvailableClientRole[]> = {
   [ParticipantTypeData.Publisher.typeName]: ['GENERATOR', 'SHARER'],
 };
 
+export const AllowedSiteRolesById: Record<number, number[]> = {
```

Comment:
I think this should be named `RecommendedSiteRolesById`, given it actually differs to the above `AllowedSiteRoles` (noting the difference with `SHARER`).

Reply:
And a second thought: is it worth removing `AllowedSiteRoles` and `GetRecommendedRoles` given they are unused?

Reply:
I removed them.